frontend/packages/metal3-plugin/src/status/host-status.ts

```diff
@@ -18,19 +14,15 @@ import { getNodeMaintenanceStatus } from './node-maintenance-status';
 export const getBareMetalHostStatus = (host: BareMetalHostKind) => {
   const operationalStatus = getHostOperationalStatus(host);
   const provisioningState = getHostProvisioningState(host);
+  const errorType = getHostErrorType(host);
 
   let hostStatus;
 
   if (operationalStatus === HOST_STATUS_ERROR) {
-    switch (provisioningState) {
-      case HOST_STATUS_REGISTERING:
-        hostStatus = HOST_STATUS_REGISTRATION_ERROR;
-        break;
-      case HOST_STATUS_PROVISIONING:
-        hostStatus = HOST_STATUS_PROVISIONING_ERROR;
-        break;
-      default:
-        hostStatus = HOST_STATUS_ERROR;
+    if (errorType) {
+        hostStatus = errorType;
```

Comment:
Nit: remove 2 spaces here, linter will complain.
```suggestion
      hostStatus = errorType;
```